services/workflows-service/docker-compose.redis.yml

```diff
@@ -0,0 +1,26 @@
+version: '3.8'
+services:
```

Comment:
Why is it located on a new docker image? Our service has to use redis in order to run ...
you can place it with a dependency and when Redis is ready start workflow service

Reply:
Why cant we use the same docker-compose-db.yml ?

we can use same like this 

```
version: '3'
services:
  db:
    image: sibedge/postgres-plv8:15.3-3.1.7
    ports:
      - ${DB_PORT}:5432
    environment:
      POSTGRES_USER: ${DB_USER}
      POSTGRES_PASSWORD: ${DB_PASSWORD}
    volumes:
      - postgres15:/var/lib/postgresql/data
  redis:
    image: redis:alpine
    ports:
      - '${REDIS_PORT}:6379'
    volumes:
      - redis-data:/data
    command: >
      --requirepass ${REDIS_PASSWORD}
      --appendonly yes
    environment:
      - REDIS_PASSWORD=${REDIS_PASSWORD}
      - REDIS_PORT=${REDIS_PORT}
volumes:
  postgres15: ~
  redis-data:
    driver: local
```

Bring only redis using :

This [Line](https://github.com/ballerine-io/ballerine/pull/2735/files#diff-65672d584fae78ff93600b1898b6b95bb50b5e51a25297cd36a840c37cd34b65R35)
can be replaced with the following

```
docker:redis: docker compose -f docker-compose.db.yml up -d redis --wait
docker:redis:down: docker compose -f docker-compose.db.yml down -v redis,
```